pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -29,7 +29,7 @@
     "amcrest" = ps: with ps; [ ha-ffmpeg ]; # missing inputs: amcrest
     "ampio" = ps: with ps; [ ]; # missing inputs: asmog
     "android_ip_webcam" = ps: with ps; [ ]; # missing inputs: pydroid-ipcam
-    "androidtv" = ps: with ps; [ ]; # missing inputs: adb-shell[async] androidtv[async] pure-python-adb
+    "androidtv" = ps: with ps; [ adb-shell ]; # missing inputs: androidtv[async] pure-python-adb
```

Comment:
Looks like this is only one of the three libraries to make that component work. You'll also need to package `androidtv` and `pure-python-adb`.

Also the regeneration of the component-packages should be in their own commit, since it touches the home-assistant attribute.

Reply:
Okay, I'll exclude this change from the PR. I also plan on packaging the other 2 libraries as well.

Reply:
It can be included in this PR, just not in the same commit.